### ISSUE TITLE
Unable to import in 0.7 Nightly

### ISSUE THREAD

**Issue by yeonsookimdev** [c42f/MicroLogging.jl#15]
```
julia> using MicroLogging
[ Info: Precompiling module MicroLogging
WARNING: could not import Base.BelowMinLevel into MicroLogging
WARNING: could not import Base.Debug into MicroLogging
WARNING: could not import Base.Info into MicroLogging
WARNING: could not import Base.Warn into MicroLogging
WARNING: could not import Base.Error into MicroLogging
WARNING: could not import Base.AboveMaxLevel into MicroLogging
WARNING: could not import Base.handle_message into MicroLogging
WARNING: could not import Base.shouldlog into MicroLogging
WARNING: could not import Base.min_enabled_level into MicroLogging
WARNING: could not import Base.catch_exceptions into MicroLogging
WARNING: could not import Base.LogState into MicroLogging
WARNING: importing deprecated binding Base.STDERR into MicroLogging.
WARNING: Base.STDERR is deprecated, use stderr instead.
 in module MicroLogging
in __init__ at C:\Users\alkorang\.julia\v0.7\MicroLogging\src\MicroLogging.jl
ERROR: InitError: UndefVarError: Info not defined
Stacktrace:
 [1] MicroLogging.InteractiveLogger(::Base.TTY) at C:\Users\alkorang\.julia\v0.7\MicroLogging\src\loggers.jl:18
 [2] __init__() at C:\Users\alkorang\.julia\v0.7\MicroLogging\src\MicroLogging.jl:54
 [3] _include_from_serialized(::String, ::Array{Any,1}) at .\loading.jl:652
 [4] _require_from_serialized(::String) at .\loading.jl:714
 [5] _require(::Base.PkgId) at .\loading.jl:1004
 [6] require(::Base.PkgId) at .\loading.jl:874
 [7] require(::Module, ::Symbol) at .\loading.jl:869
during initialization of module MicroLogging

julia> versioninfo()
Julia Version 0.7.0-DEV.5111
Commit 6291d3eb4b* (2018-05-16 05:58 UTC)
Platform Info:
  OS: Windows (i686-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-3630QM CPU @ 2.40GHz
  WORD_SIZE: 32
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, ivybridge)

julia>
```

Does `MicroLogging` conflict with `CoreLogging` in `0.7.0`?

**Comment by JoshChristie** [comment]
I believe `MicroLogging` was included in Base julia-0.7 by default. The module in Base is called `CoreLogging`. Hope that helps.

**Comment by yeonsookimdev** [comment]
Yes, from 0.7 Julia has `CoreLogging`.
